### ISSUE TITLE
VS Code unable to recognize any configured extension settings

### ISSUE THREAD

**Issue by prmichaelsen** [microsoft/vscode#231760]
Does this issue occur when all extensions are disabled?: Yes/No

VS Code

Version: 1.94.2
Commit: 384ff7382de624fb94dbaf6da11977bba1ecd427
Date: 2024-10-09T16:08:44.566Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Darwin arm64 23.6.0

OS 

NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2:-:internal"
HOME_URL="https://amazonlinux.com/"
VARIANT="internal"

None of my extensions are able to load any configuration values and config values in the settings.json are "greyed out" as if VS Code cannot recognize them. This issue has been occurring even after trying different VS Code version.s

Steps to Reproduce:

1. Not sure

![Image](https://github.com/user-attachments/assets/bc83573d-ee4e-47c3-a133-4ec1031b4a7e)

**Comment by rzhao271** [comment]
Does any hover content show up when you hover over a setting in that file?
Also, are you in a non-default [profile](https://code.visualstudio.com/docs/editor/profiles)? If so, you might be viewing settings that do not apply to the current profile, and could view the settings that do apply to the current profile by running the "Preferences: Open User Settings (JSON)" command.

**Comment by prmichaelsen** [comment]
For hovering, I see
![Image](https://github.com/user-attachments/assets/cb6e348f-61a1-439e-8c53-ec722b5b622d)

I'll look into the profiles

**Comment by prmichaelsen** [comment]
Looks like I am using the Default profile. I accessed the settings file via Cmd + Shift + P > Open User Settings

**Comment by rzhao271** [comment]
Do the extensions themselves load in properly?

**Comment by prmichaelsen** [comment]
Yes, I also tried an extension bisect but all extension settings were "greyed out" for the entire bisect process.

**Comment by prmichaelsen** [comment]
Sorry to be a squeak wheel here, is there any chance someone can take another look at this? Are there any more troubleshooting steps I could take?

**Comment by rzhao271** [comment]
Could you try moving out the data folders listed under the [clean uninstall instructions](https://code.visualstudio.com/docs/setup/uninstall#_clean-uninstall) and re-launching VS Code to see whether starting from a fresh environment works?

**Comment by prmichaelsen** [comment]
I'll give that shot, thanks
